### DISCUSSION TITLE
What I Want to Build in Clojure

### DISCUSSION THREAD

**Discussion by avidrucker** [athensresearch/athens#1226] [General]
As learners of Clojure, the sky is the limit in terms of what we can aim to accomplish and build. I'd love to hear what are any projects you have in mind as part of your learning journey. If there is enough interest in a particular project, we may do some pair programming or group work.

**Comment by avidrucker** [comment]
To-Do List App

**Comment by avidrucker** [comment]
Microservice

**Comment by avidrucker** [comment]
Microchip Controller

**Reply by Ejipuh** [reply]
Arduino?

**Reply by Ejipuh** [reply]
This would be fun to sync with Athens. You could make your own custom controllers with it, or something like that.

**Comment by avidrucker** [comment]
Browser Plug-in / Add-on

**Comment by avidrucker** [comment]
A real-time browser or command line game such as Snake

**Comment by avidrucker** [comment]
A data analytics pipeline

**Comment by avidrucker** [comment]
A machine learning model implementation and demonstration on real or fake data

**Comment by avidrucker** [comment]
A robot

**Comment by avidrucker** [comment]
A chat bot

**Comment by avidrucker** [comment]
An Android or iOS native application

**Comment by avidrucker** [comment]
A Natural Language Processing model and/or application

**Comment by Ejipuh** [comment]
I would really like to build or find a tool like [Glamorous Toolkit](https://gtoolkit.com/), that helped me to read and understand code systems better. If things went well, this could become an onboarding tool to learn Clojure with, and could influence the way that Athens is used.
